plugins/modules/efs.py

```diff
@@ -502,6 +527,30 @@ def update_lifecycle_policy(self, name, transition_to_ia):
                 changed = True
         return changed
 
+    def change_backup_policy(self, name, status):
+        """
+        Enable/disable automatic backup policy of the FileSystem.
+        """
+        changed = False
+        state = self.get_file_system_state(name)
+        if state in [self.STATE_AVAILABLE, self.STATE_CREATING]:
```

Comment:
If `state` doesn't match the condition, we silently ignore it and return "unchanged" even if the value is not the one expected.

Reply:
Not sure I understand what you mean by this, isn't the "unchanged" when the `changed= False`?